### ISSUE TITLE
[SSD/PyTorch] Question: How to Compute Validation Loss ?

### ISSUE THREAD

**Issue by Venketaramana** [NVIDIA/DeepLearningExamples#1035]
I need some advise on what approach to take to compute `validation loss` for this pipeline. I am training on a custom dataset and I am getting some good results, however there is a particular class that is performing poorly. I would like to analyze the validation performance, but I can't seem to figure out how to compute the `validation loss` for this pipeline.
